dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -35,10 +38,8 @@
 
     @Override
     public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGitHubUtil)
```

Comment:
This method has quite a bit of logic in it.  Lets break it up into some smaller helper methods.